Source/Charts/Data/Implementations/Standard/BarChartDataSet.swift

```diff
@@ -29,7 +29,7 @@ open class BarChartDataSet: BarLineScatterCandleBubbleChartDataSet, BarChartData
         initialize()
     }
     
-    public override init(values: [ChartDataEntry]?, label: String?)
+    public override init(values: [ChartDataEntry], label: String)
```

Comment:
I'm ok to remove, but what's the reason removing all values and labels optional?

Reply:
Why would they be optional? We already have an initializer that doesn't take a `label`, and backing stores shouldn't be optional.

Reply:
ok.